pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -44,6 +44,9 @@ Base.setproperty!(x::MPtr, f::Union{AbstractString,Int64}, v) = Globals.ASS_REC(
 Base.zero(x::GAPInputType_internal) = Globals.ZERO(x)
 Base.one(x::GAPInputType_internal) = Globals.ONE(x)
 Base.:-(x::GAPInputType_internal) = Globals.AINV(x)
+Base.:-(x::Int64,y::GAP.MPtr) = GAP.julia_to_gap(x)-y
+Base.:*(x::GAP.MPtr,y::String) = x*GAP.julia_to_gap(y)
+Base.getindex(x::GAP.MPtr,y::String) = x*GAP.julia_to_gap(y)
```

Comment:
This `getindex` function is not general, cause it will prevent us to use `[]` with Julia strings. You should remove it fromthis  and implement `[]` on the GAP side.

Reply:
OK.

Reply:
I have this on the GAP side

https://github.com/homalg-project/homalg_project/blob/master/MatricesForHomalg/gap/Julia.gi#L15

but it is apparently not enough. But I see, that the RHS of the assignment is far too special.

Reply:
Maybe the solution is adding a function like this one: https://github.com/oscar-system/GAP.jl/blob/master/pkg/GAPJulia/JuliaInterface/julia/gap.jl#L23 but which takes arbitrary second arguments, or `String`s for example

Reply:
```
Base.getindex(x::GAP.MPtr,y::String) = Globals.ELM_LIST(x,GAP.julia_to_gap(y))
```
If it is OK for you I can use it.